NOTICE

```diff
@@ -3,3 +3,6 @@ Copyright 2017 The Apache Software Foundation.
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+This product includes software from the
```

Comment:
Actually, we may need to include something in the source. Have a look at src/pdxautoserializer/impl/CPPParser/CPPDictionary.hpp.

I think we need to remove the Apache header and add this code to the public domain list.